content/swiftui/concepts/views/terms/button/button.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: 'Button'
+Description: 'A very basic component in UI, which is used to interact and perform certain actions'
+Subjects:
+  - 'Sotfware Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Buttons'
```

Comment:
"Buttons" is not listed in the Tags document. This should either be removed, or you should add "Buttons" to https://github.com/Codecademy/docs/blob/main/documentation/tags.md

Reply:
I have added buttons tag in the tags.md